js/App.js

```diff
@@ -125,11 +127,33 @@ App.prototype.setUnreadChatMessageCount = function(count) {
   }
 };
 
-App.prototype.intlNumFormat = function(numberToFormat, maxDigits){
-  maxDigits = maxDigits || 8; //default to show down to the satoshi (.00000001)
+App.prototype.intlNumFormat = function(numberToFormat, maxDigits) {
+  if (maxDigits === undefined) { // Allow maxDigits = 0
+    maxDigits = 8; //default to show down to the satoshi (.00000001)
+  }
   return new Intl.NumberFormat(window.lang, {maximumFractionDigits: maxDigits}).format(numberToFormat);
 };
 
+/**
+ * Format a bitcoin amount in the user's locale.
+ *
+ * @param {Number} amount - Bitcoin (BTC) amount.
+ * @param {Number} [maxDigits=8] - Maximum number of fraction digits to display.
+ * @returns {String} localised amount ending with ' BTC'
+ *
+ * @see intlNumFormat
+ */
+App.prototype.formatBitcoin = function(amount, maxDigits) {
+  if (_app.bitcoinUnit === 'mBTC') {
```

Comment:
Since this is an instance method, would you please user `this` instead of `_app` to refer to the app instance?